packages/card/src/__tests__/Card.spec.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import { shallow } from 'enzyme';
+import Card from '../Card';
+import cardExamples from '../__demo__/examples/cardIndex';
+import cardActionsExamples from '../__demo__/examples/cardActionsIndex';
+import cardBlockExamples from '../__demo__/examples/cardBlockIndex';
+import cardImageExamples from '../__demo__/examples/cardImageIndex';
+import cardTitleExamples from '../__demo__/examples/cardTitleIndex';
+import testDemoExamples from '../../../../utils/test/testDemoExamples';
+
+const examples = [].concat(
```

Comment:
How about just add an `index.js` to the examples directory like other components, then import that here?